docs/docs/legal/vulnerability-disclosure-policy.mdx

```diff
@@ -0,0 +1,90 @@
+
+You must not do research or testing that involves
+
+- Any activity that violates applicable law
+- Modify or destroy any data that does not belong to you
+- Accessing or attempt to access data that does not belong to you
+- Executing denial of service attacks
+- Executing load testing
+
+Exceptions may be granted after your initial report by a member of our security team.
+
+## Reporting a vulnerability
+
+To file an incident, please disclose it by e-mail to [security@zitadel.com](mailto:security@zitadel.com) including the following details of the vulnerability:
+
+- Target: ZITADEL, Website (zitadel.com), ZITADEL Cloud (zitadel.cloud), Other (please describe)
+- Type: For example DoS, authentication bypass, information disclosure, broken authorization, ...
+- Description: Provide a detailed explanation of the issue, steps to reproduce, and assumptions you have made
+- Url / Location (optional): The url of the vulnerability
+- Contact details (optional): In case we should contact you on a different channel
+
+At the moment GPG encryption is no yet supported, however you may sign your message at will.
+
+Your email will be acknowledged within 48 hours.
+We will follow-up within the next 3 business days indicating next steps in handling your report.
+
+If you haven't received a response within 48 hours, or you didn't get a reply from our security team within the last 5 days, please contact [hi@zitadel.com](mailto:hi@zitadel.com).
+
+Please inform us in your report whether we should mention your contribution.
+We will not publish this information by default to protect your privacy.
+
+### What not to report
+
+- Disclosure of known public files or directories, e.g. robots.txt, files under .well-known, or files that are included in our public repositories (eg, go.mod)
+- DoS of users when [Lockout Policy is enabled](https://zitadel.com/docs/guides/manage/console/instance-settings#lockout)
+- Suggestions on Certificate Authority Authorization (CAA) rules
+- Suggestions on DMARC settings
+
+## Disclosure Process
+
+Our security team will follow the disclosure process:
+
+1. We will acknowledge the receipt of your vulnerability report
+2. Our security team will try to verify, reproduce, and determine the impact of your report
+3. A member of our team will respond to either confirm or reject your report, including an explanation
+4. Code will be audited to assess if the report uncovers similar issues
+5. Fixes are prepared for the latest release
+6. On the date that the fixes are applied, we will publish a [security advisory](https://github.com/zitadel/zitadel/security/advisories). Affected users of our Product, Services, or Website will be informed of the fix and required actions.
+
+We think it is crucial to publish advisories `ASAP` as mitigations are ready. But due to the unknown nature of the disclosures the time frame can range from 7 to 90 days.
+
+## Bug Bounty / Compensation
```

Comment:
We could credit the CVE to the person publicly if that's agreed

Reply:
That's more the case for public services, right? This is being mentioned in SECURITY.md
Do you think its worth re-stating it also on this policy?

Reply:
In the reporting process. Can re-iterate here.